app/views/registers/show.html.haml

```diff
@@ -37,25 +36,25 @@
           %p{class: 'govuk-!-margin-bottom-0'} Always get the most up-to-date data in this register in JSON format.
 
     %section.govuk-grid-row#records_wrapper{ data: { module: 'accordion-with-descriptions'}}
-      .govuk-grid-column-full.subsection__header#search_wrapper
-        %h2.govuk-heading-m.subsection__title
-          About the data
-      .govuk-grid-column-two-thirds.subsection__content.js-accordion-subsection#about-the-data__0
-        %p This register has #{number_with_delimiter(@register_records_total_count)} records and #{@register.register_fields.count} fields
-      .govuk-grid-column-one-third.subsection__content.js-accordion-subsection#about-the-data__1
-        = form_tag register_path(@register.slug, anchor: 'records_wrapper'), method: :get, id: 'search_records', class: 'records-search' do
-          = label_tag 'Search', nil, for: 'q', class: 'visually-hidden'
-          = search_field_tag 'q', nil, class: 'search-input', placeholder: 'Search', value: @search_term, autocomplete: 'off'
-          = submit_tag 'Search', name: nil, class: 'search-submit'
-      .govuk-grid-column-full.subsection__content.js-accordion-subsection#about-the-data__2
+      .govuk-grid-column-full
+        %hr{class: 'govuk-!-margin-top-0'}
+        %h2.govuk-heading-m About the data
+      %div{class: 'govuk-grid-column-two-thirds govuk-!-margin-bottom-3'}
+        %p
+          There are
+          <!-- -->
```

Comment:
can we remove these comment blocks?

Reply:
Same as the other instance of this - the comment blocks preserve a space between the words.